ils_middleware/tasks/amazon/sqs.py

```diff
@@ -35,6 +35,15 @@ def get_group(resource_uri: str) -> str:
     return f"{resource_uri} returned error {result.status_code}"
 
 
+def get_edit_groups(resource_uris: list) -> list:
+    """Retrieves the Resource's editGroups."""
+    edit_groups = []
+    for resource_uri in resource_uris:
+        edit_groups.append(get_group(resource_uri))
```

Comment:
If `get_group()` returns an error (line 35), will it be appended to `edit_groups` and, if so, is that OK?